packages/scandipwa/src/component/CartItemPrice/CartItemPrice.container.js

```diff
@@ -30,17 +30,23 @@ export const mapDispatchToProps = () => ({});
 export class CartItemPriceContainer extends PureComponent {
     static propTypes = {
         getCartItemPrice: PropTypes.func.isRequired,
-        getCartItemSubPrice: PropTypes.func.isRequired
+        getCartItemSubPrice: PropTypes.func.isRequired,
+        currency_code: PropTypes.string.isRequired,
+        mix: PropTypes.object.isRequired
```

Comment:
`PropTypes.object` is deprecated, as far as I remember, maybe we should use `PropTypes.shape` instead?